futurex_openedx_extensions/dashboard/statistics/courses.py

```diff
@@ -32,9 +32,9 @@ def get_courses_count(
         fx_permission_info, visible_filter=visible_filter, active_filter=active_filter
     )
 
-    return q_set.values('org').annotate(
+    return q_set.values(org_upper_case=Upper('org')).annotate(
         courses_count=Count('id')
-    ).order_by('org')
+    ).order_by(Upper('org'))
```

Comment:
To match domain names, what do you think?

```suggestion
    ).order_by(Lower('org'))
```

Reply:
done